darwin/future/core/items/assign_items.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from darwin.future.core.client import ClientCore
+from darwin.future.core.types.common import JSONType
+from darwin.future.data_objects.typing import UnknownType
+
+
+def assign_items(
+    client: ClientCore,
+    team_slug: str,
+    dataset_ids: int | list[int],
+    assignee_id: int,
+    workflow_id: str,
+    filters: dict[str, UnknownType],
```

Comment:
we have a datatype for this, JSONDict, which is basically a wrapper for dict[str, Any]. Nothing huge but good to keep consistent. It's in future data_objects I think?

Reply:
This is copy-pasted from the set_stage, but sure, I'll change it here.